src/sinks/datadog/metrics/encoder.rs

```diff
@@ -882,15 +1013,16 @@ mod tests {
     };
 
     use super::{
```

Comment:
💬 Is there value in adding corresponding tests here for the Series V2 logic? From my understanding, these test currently only exercise the Sketch and V1 Series logic.

Reply:
There are a couple that are clear cases for extending to the v2, such as the `incorrect_metric_for_endpoint_causes_error` and `encode_single_series_metric_with_default_limits`.

However its less clear to me the value of extending the existing test cases that cover the payload size limits etc. Curious if @tobz  has opinions there.

(Side note ofc, the integration test has been extended to cover the v2 case).

Reply:
btw, I already started extending these, even more than the ones I listed